src/Makefile.am

```diff
@@ -2,11 +2,11 @@
 sbin_PROGRAMS = forked-daapd
 
 if COND_ITUNES
-ITUNES_SRC=filescanner_itunes.c
+ITUNES_SRC=library/filescanner_itunes.c
 endif
 
 if COND_SPOTIFY
-SPOTIFY_SRC=spotify.c spotify.h
+SPOTIFY_SRC=spotify.c spotify.h spotify_webapi.c spotify_webapi.h
```

Comment:
The webapi-files only help access the spotify webapi, the real library source is in spotify.c. I did not move spotify.c (+ *_webapi) because they do more than only filling the library.